docs/source/building.md

```diff
@@ -25,10 +25,6 @@ When configuring your project just set:
   -DKokkos_ROOT=${kokkos_install_prefix} \
   -DCMAKE_CXX_COMPILER=${compiler_used_to_build_kokkos}
 ````
-Note: You may need the following if using some versions of CMake (e.g. 3.12):
-````cmake
-cmake_policy(SET CMP0074 NEW)
-````
```

Comment:
Note: Using `Kokkos_ROOT` requires [CMP0074](https://cmake.org/cmake/help/latest/policy/CMP0074.html) to be set to `NEW`, e.g., using
````cmake
cmake_policy(SET CMP0074 NEW)
````
when the minimum required CMake version in the dowsntream project is less than 3.12.